pkgs/development/python-modules/fluidasserts/overlays/mixpanel.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+
+# Python Dependencies
+, six
+}:
+
+buildPythonPackage rec {
+  pname = "mixpanel";
+  version = "4.5.0";
+
```

Comment:
why not just add this directly to python-modules?

Reply:
this is already in python modules, but in a **very** deprecated version one that only works in python2, and many python packages depend on it

it breaks 0 things this way